corehq/apps/es/case_search.py

```diff
@@ -233,7 +233,7 @@ def multiplex_to_adapter(domain):
     return None
 
 
-def case_property_query(case_property_name, value, fuzzy=False, multivalue_mode=None):
+def case_property_query(case_property_name, value, fuzzy=False, multivalue_mode=None, boost_first=False):
```

Comment:
@esoergel I am feeling a bit iffy about adding yet another flag here. I see two alternatives:
1) just create a separate function `case_property_query_boost_first` and call it from `fuzzy_date`
2) use `_base_property_query` directly in `fuzzy_date` probably renaming it without the underscore in the process.
I am curious, if you have a preference?